### DISCUSSION TITLE
Simulate a 1D-Lidar (laser scan)

### DISCUSSION THREAD

**Discussion by JiayunjieJYJ** [bulletphysics/bullet3#4052] [Q&A]
Can I simulate a 1D-Lidar (laser scan) by using the output of depth data of getCameraImage()? In other words, I want to get a horizonal distance(depth) data like a 1D-Lidar.
  

I make it in GUI mode , but I find in DIRECT mode, the depth image does not work, it cant detect objects!  Anybody know how to solve it???

**Comment by rohit-kumar-j** [comment]
Provide your code for the lidar part.
